app/src/main/java/com/sun/colornotetaking/data/local/dao/TaskDAO.java

```diff
@@ -10,6 +10,8 @@
 
     List<Task> getTasks();
 
+    List<Task> getListTasksById(List<Integer> tasksId);
```

Comment:
```suggestion
    List<Task> getTasks(List<Integer> tasksId);
```
nên dùng tính chất đa hình